miden-lib/asm/miden/tx.masm

```diff
@@ -53,39 +53,39 @@ export.get_output_notes_hash
     # => [COM]
 end
 
-#! Creates a new note and returns a pointer to the memory address at which the note is stored.
+#! Creates a new note and returns the index of the note.
 #!
 #! Inputs: [tag, note_type, RECIPIENT]
-#! Outputs: [note_ptr]
+#! Outputs: [note_idx]
 #!
 #! tag is the tag to be included in the note.
 #! note_type is the storage type of the note
 #! RECIPIENT is the recipient of the note.
-#! note_ptr is the pointer to the memory address at which the note is stored.
+#! note_idx is the index of the the note to which the asset is added.
 export.create_note
```

Comment:
The comment on line 64 doesn't seem to be right (i.e., there should be no references to "asset" here).